fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/model/BulkExportJobRestTest.java

```diff
@@ -35,175 +34,177 @@ public void testBulkExportJobInstanceResponseBuilderAndGenerator() throws IOExce
-            
+
             JSONAssert.assertEquals(expected, str, false);
         } catch(Exception e) {
             e.printStackTrace();
             fail("failed to parse", e);
         }
     }
-    
+
     @Test
     public void testBulkImportJobResponse() {
-        String jsonString = "{\n" + 
-                "    \"jobName\": \"BulkImportJob\",\n" + 
-                "    \"instanceId\": 2,\n" + 
-                "    \"appName\": \"fhir-bulkimportexport-webapp#fhir-bulkimportexport-webapp-4.0.0-SNAPSHOT.war\",\n" + 
-                "    \"submitter\": \"fhiradmin\",\n" + 
-                "    \"batchStatus\": \"STARTING\",\n" + 
-                "    \"jobXMLName\": \"FhirBulkExportChunkJob\",\n" + 
-                "    \"instanceState\": \"SUBMITTED\",\n" + 
-                "    \"lastUpdatedTime\": \"2019/09/13 07:38:07.159 -0400\",\n" + 
-                "    \"_links\": [\n" + 
-                "        {\n" + 
-                "            \"rel\": \"self\",\n" + 
-                "            \"href\": \"https://localhost:9443/ibm/api/batch/jobinstances/2\"\n" + 
-                "        },\n" + 
-                "        {\n" + 
-                "            \"rel\": \"job logs\",\n" + 
-                "            \"href\": \"https://localhost:9443/ibm/api/batch/jobinstances/2/joblogs\"\n" + 
-                "        }\n" + 
-                "    ]\n" + 
+        String jsonString = "{\n" +
+                "    \"jobName\": \"BulkImportJob\",\n" +
+                "    \"instanceId\": 2,\n" +
+                "    \"appName\": \"fhir-bulkimportexport-webapp#fhir-bulkimportexport-webapp-4.0.0-SNAPSHOT.war\",\n" +
+                "    \"submitter\": \"fhiradmin\",\n" +
+                "    \"batchStatus\": \"STARTING\",\n" +
+                "    \"jobXMLName\": \"FhirBulkExportChunkJob\",\n" +
+                "    \"instanceState\": \"SUBMITTED\",\n" +
+                "    \"lastUpdatedTime\": \"2019/09/13 07:38:07.159 -0400\",\n" +
+                "    \"_links\": [\n" +
+                "        {\n" +
+                "            \"rel\": \"self\",\n" +
+                "            \"href\": \"https://localhost:9443/ibm/api/batch/jobinstances/2\"\n" +
+                "        },\n" +
+                "        {\n" +
+                "            \"rel\": \"job logs\",\n" +
+                "            \"href\": \"https://localhost:9443/ibm/api/batch/jobinstances/2/joblogs\"\n" +
+                "        }\n" +
+                "    ]\n" +
                 "}";
-        
-        
+
+
         try(InputStream in = new ByteArrayInputStream(jsonString.getBytes())){
             BulkExportJobInstanceResponse parsedObj = BulkExportJobInstanceResponse.Parser.parse(in);
-            
+
             String str = BulkExportJobInstanceResponse.Writer.generate(parsedObj);
             JSONAssert.assertEquals(jsonString, str, false);
-            
+
             System.out.println(str);
```

Comment:
let's use a logger, rather than output the str, that way we can turn it on / off

Reply:
will just removed it, should be very easy to get the value during debugging.